src/main/js/custom_commands/removePageHead.js

```diff
@@ -1,13 +1,13 @@
-/**
- * Nightwatch command to remove the breadcrumb bar on classic jenkins pages.
+/** @module removePageHead
```

Comment:
Actually if you compare the page_objects there all start like this, I only added it since @tfennelly forgot it. ;)